pkg/koordlet/resmanager/cpu_burst.go

```diff
@@ -198,6 +198,12 @@ func (b *CPUBurst) start() {
 			// ignore non-burstable pod, e.g. LSR, BE pods
 			continue
 		}
+		if podMeta.Pod.Status.Phase == corev1.PodSucceeded {
```

Comment:
How about other phases like `corev1.PodFailed`? We may check if the phase is not Running or Pending.

Reply:
> How about other phases like `corev1.PodFailed`? We may check if the phase is not Running or Pending.

Because only successfully exited pods are sure never to be restarted.
![kubernetes-pod-life-cycle](https://user-images.githubusercontent.com/32814765/199147909-9e79df0c-44fb-472e-b099-1d5c90e1b88c.jpeg)

Reply:
What if it's restart policy is set to never?

Reply:
'PodFailed' means all containers have terminated, so the koordlet would fail to update any of the container-level cgroups without correct cgroup paths.

Reply:
In fact, the container cgroup file is only available as long as the container exists.

Reply:
The comments indicate that `Pending` pods may have some containers running.  
e.g. a pod with part of the init containers started, a pod with part of regular containers terminated.

Reply:
okay, I'll change the code then.